js/tiles.js

```diff
@@ -383,6 +383,10 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/banner-Chess.svg");
         },
         visit: function() {
+            if(player.inventory.has('Key')){
+                player.inventory.remove('Key')
+                alertInfo('Oops lost your key!')
```

Comment:
Should we have two alerts? Can you modify the alerts similar to something like this:

```javascript
if(player.inventory.has('Key')){
  alertInfo("Checkmate", 'Oops lost your key! Come on, Try to get out from here.')
} else {
  alertInfo("Checkmate", Come on, Try to get out from here.')
}
```

So that we will only have one alert. What do you say?

Reply:
@arjunpandey2003 can you make,
`alertInfo('Oops lost your key!');`
to
`alertInfo("Checkmate", 'Oops lost your key! Come on, Try to get out from here.');

so that we won't loose the checkmate functionality as well? What do you say?

besides that pls lower the if block after loading the tiles if possiblr for a better view. We get in and then display the alert.